### PR TITLE
Changes to GEOMAR boot camp page.

### PR DESCRIPTION
**Pull request by amyrhoda**
Removed reference to _content.html, added HTML for their planned
syllabus. Also added Germany to location because I didn't know where
Kiel was. (Kept wanting it to be typo of Kiev.)

### CODE REVIEW

File: bootcamps/2013-05-geomar.html [hunk 1]
```diff
@@ -7,11 +7,70 @@
...
-{% include "_who.html" %}
 {% include "_requirements.html" %}
-{% include "_content.html" %}
+
+<p>
+  <strong>Content:</strong> On the first day of the bootcamp we will cover:
+</p>
+<ul>
+  <li>Unix shell</li>
+  <ul>
```
Comment (gvwilson) [inline bootcamps/2013-05-geomar.html]: Strictly speaking, you can't put a `<ul>` directly in a `<ul>`: it has to be inside one of the `<li>`'s, like this:

``` html
<ul>
  <li>First major
    <ul>
      <li>First minor</li>
      <li>Second minor</li>
    </ul>
  </li>
  <li>Second major
    <ul>
      <li>Third minor</li>
    </ul>
  </li>
</ul>
```

Comment (amyrhoda) [inline bootcamps/2013-05-geomar.html]: Huh. I tried that and it looks like this: 
dbleList

Note goofy double bullets. Am I doing something wrong? Here's my HTML:

```
On the second day we will cover:
</p>
<ul>
  <li>Python</li>
  <li>
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
```

Comment (jiffyclub) [inline bootcamps/2013-05-geomar.html]: You want this:

``` html
<ul>
  <li>Python
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
   </ul>
  </li>
</ul>
```

Which renders to:

<ul>
  <li>Python
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
   </ul>
  </li>
</ul>


Note where the closing `</li>` element occurs for the Python item.

**Comment by amyrhoda** [comment]
Oooh! Duh. Thanks.

**Pull request merged**